azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -300,6 +300,8 @@ azkaban.FlowExecuteDialogView = Backbone.View.extend({
         if (node.type == "flow" && statusNode.type == "flow") {
           this.assignInitialStatus(node, statusNode);
         }
+      } else {
```

Comment:
This function (`assignInitialStatus`) traverses the jobs in the flow and looks up the same job in original flow to assign an initial status. If that lookup fails, it means that a new job has been added to the flow since the original execution. In that case there is no initial status to copy from the original execution, so we set `node.missing=true` to mark that.

I don't know if adding a comment in the code is the best way to improve. How about I rename this flag? For example, `node.noInitialStatus` or `node.notPresentInOriginalExecution`. What would you prefer?